main.js

```diff
@@ -0,0 +1,165 @@
+
+var setDefaultLocation = function() {
+  //if zipcode, store the zipcode
+  //if no zipcode, store the lat/long
+  var defaultValues = {};
+  if (searchLocation.zip){
+    defaultValues.zip = searchLocation.zip;
+  } else {
+    defaultValues.latitude = searchLocation.latitude;
+    defaultValues.longitude = searchLocation.longitude;
+  }
+  var defaultLocation = JSON.stringify(defaultValues);
+  localStorage.defaultLocation = defaultLocation;
```

Comment:
While this works, it would be nicer to specify window.localStorage to make sure it's clear that it is not a local variable.

Reply:
Also when adding things into local storage, get in the habit of naming them less generically.  Keep in mind that all the pages on this site would use the same storage, so if another page had a default location in a different format, you'd run into issues.